packages/ds-medicare-gov/src/styles/components/_override.help_drawer.scss

```diff
@@ -2,31 +2,15 @@
 $mct-drawer-close-button-size: 26px;
 $mct-drawer-heading-font-size: $h1-mobile-font-size;
 
-.mct-c-help-drawer {
-  .ds-c-drawer__header {
-    background-color: $drawer-header__background-color;
-  }
-
-  .ds-c-drawer__header-heading {
-    font-family: $font-family-montserrat;
-    font-size: $mct-drawer-heading-font-size;
-    font-weight: $font-weight-bold;
-    text-align: left;
-  }
-
-  .ds-c-drawer__close-button {
-    border: 0;
-    color: $drawer-close__color;
-
-    svg {
-      height: $mct-drawer-close-button-size;
-      width: $mct-drawer-close-button-size;
-    }
+.ds-c-drawer__header-heading {
+  font-family: $font-family-montserrat;
+  font-size: $font-size-xl;
+  font-weight: $font-weight-bold;
+  line-height: $font-line-height-base;
+}
 
-    .ds-c-help-drawer__footer {
-      display: none;
```

Comment:
As stated before, mgov doesn't seem to use the footer prop (as evident with this style). That's why I added that conditional logic to remove the markup if the props weren't used.